students/Lincoln_Zhang/Session2/grid_print.py

```diff
@@ -0,0 +1,53 @@
+#
+#Session 2 Class excercise
+#print grid like this
+#+ - - - - + - - - - +
+#|         |         |
+#|         |         |
+#|         |         |
+#|         |         |
+#+ - - - - + - - - - +
+#|         |         |
+#|         |         |
+#|         |         |
+#|         |         |
+#+ - - - - + - - - - +
+
+def print_plus():
+	print ("+",end="")
+
+def print_minus():
+	print(" - - - - ",end="")
+
+def print_space():
+	print("         ",end="")
+
+def print_pipe():
+	print("|",end="")
+
+def print_newline():
+	print("\n")
+
+def grid_print(r):
+	for r in range(r):
+		if r%5 == 0:
```

Comment:
This print only prints the top half of the grid, you will want to have a grid of 4 squares at the end.